src/rust/catnap/win/config.rs

```diff
@@ -0,0 +1,125 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+//======================================================================================================================
+// Imports
+//======================================================================================================================
+
+use crate::{
+    demikernel::config::Config,
+    runtime::fail::Fail,
+};
+use ::yaml_rust::Yaml;
+use std::{
+    ops::{
+        Fn,
+        Index,
+    },
+    time::Duration,
+};
+use windows::Win32::Networking::WinSock::tcp_keepalive;
+
+//======================================================================================================================
+// Constants
+//======================================================================================================================
+
+/// Name for the libos in configs.
+const LIBOS: &str = "catnap";
+
+//======================================================================================================================
+// Associated Functions
+//======================================================================================================================
+
+/// Windows-specific configuration for Demikernel configuration object.
+impl Config {
```

Comment:
We should allow overriding these values via env vars. This will make it easier to run inside the CI infrastructure.